pwndbg/elf.py

```diff
@@ -68,6 +84,95 @@ def read(typ, address, blob=None):
     return obj
 
 
+@pwndbg.memoize.forever
```

Comment:
Can we change it to memoize on new objfile?

Reply:
Done